wasm-engines/docker/wasm-engines-bench.Dockerfile

```
@@ -9,6 +9,7 @@ FROM ewasm/wavm:1 AS wavm
 FROM ewasm/fizzy:1 AS fizzy
 FROM ewasm/asmble:1 AS asmble
 FROM ewasm/wasmi:1 AS wasmi
+FROM ewasm/eos-vm:1 AS eos
```

```suggestion
FROM ewasm/eos-vm:1 AS eosvm
```

I'd also just call all the files `eosvm`.